src/collections/_documentation/workflow/alerts-notifications/index.md

```diff
@@ -7,7 +7,9 @@ Sentry notifications are broadly categorized into alerts and non-alert notificat
 
 ## Alerts
 
-Sentry provides both Metric Alerts and Issue Alerts. You define either type of alert per project.
+Issue Alerts warn you about problematic _individual Sentry Issues_. An issue alert fires once for every issue that matches its conditions.
```

Comment:
let's caps the "I": " ... once for every **Issue** that matches ..."

Reply:
To emphasize the word, I'd rather italicize it.

Reply:
Updated with italics.

Reply:
it wasn't for emphasis, we're trying to make it more of a proper noun, i.e. a specific sentry concept

Reply:
If that's the case, we want to keep things consistent and capitalize **all** instances of the word "issue."

Reply:
Is that what we want? If so, I recommend a separate PR for that.

Reply:
yeah agree we should. i was thinking since this is an important paragraph we could just make it here, but if you think the inconsistency isn't worth it that's totally fine

Reply:
👍